### ISSUE TITLE
Internal links with dashes not detected

### ISSUE THREAD

**Issue by manunamz** [maximevaillancourt/digital-garden-jekyll-template#47]
<img width="338" alt="Screen Shot 2021-02-12 at 11 04 16 AM" src="https://user-images.githubusercontent.com/75578970/107791624-2a3eb380-6d22-11eb-9675-47ba5313d54e.png">

**Comment by paulbricman** [comment]
Ran into the same issue.

https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/master/_plugins/bidirectional_links_generator.rb

Change:
```
).gsub('_', ' ').gsub('-', ' ').capitalize
```
Into:
```
).gsub('_', ' ').capitalize
```

This solved it for me, and nothing else seems to break!

**Comment by meewgumi** [comment]
This fix worked for me too! Hope this demo helps @manunamz 

- [Demo Page](https://garden.megu.space/site-organization.html#note-naming) - heading "Note Naming"
- [bidirectional_links_generator.rb](https://github.com/meewgumi/digital-garden-apache-template/blob/main/_plugins/bidirectional_links_generator.rb)

**Comment by maximevaillancourt** [comment]
Fixed in 8544fd4f36d4d46290b3090c68fb39df87393e00. Thanks everyone! :bow:

**Comment by maximevaillancourt** [comment]
Tried to fix it too quickly. Turns out, that doesn't fix all cases. Re-opening, I'll take a look at another time.

**Comment by meewgumi** [comment]
@maximevaillancourt I forgot to mention that after I pushed this fix I had to go re-create wikilinks for "your-first-note" throughout the site. I ended up just writing the link on the index page [in HTML](https://raw.githubusercontent.com/meewgumi/digital-garden-apache-template/main/index.md)

**Comment by manunamz** [comment]
Thank you good sir! 🥳

**Comment by maximevaillancourt** [comment]
Fixed in #103 :heavy_check_mark:
